pkgs/development/compilers/blueprint/default.nix

```
@@ -1,46 +1,63 @@
-{ python3
+{ gtk4
+, python310
+, python310Packages
 , stdenv
 , fetchFromGitLab
 , gobject-introspection
 , lib
 , meson
 , ninja
+, testers
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "blueprint-compiler";
-  version = "0.2.0";
+  version = "0.6.0";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "jwestman";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LXZ6n1oCbPa0taVbUZf52mGECrzXIcF8EaMVJ30rMtc=";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "blueprint-compiler";
```

adding a suffix to pname should not change this